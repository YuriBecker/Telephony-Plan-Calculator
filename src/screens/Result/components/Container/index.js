import React from 'react';
import { View } from 'react-native';
import PropTypes from 'prop-types';
import colors from '../../../../styles/colors';

const index = ({ children }) => {
  return (
    <View
      style={{
        flex: 1,
        justifyContent: 'space-around',
        alignItems: 'center',
        backgroundColor: colors.black[1],
        padding: 30,
      }}
    >
      {children}
    </View>
  );
};

export default index;

index.propTypes = {
  children: PropTypes.node.isRequired,
};
