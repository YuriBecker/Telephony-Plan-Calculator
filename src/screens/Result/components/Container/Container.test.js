import React from 'react';
import { shallow } from 'enzyme';

import Container from './index';

describe('Testing Container component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Container>Some data</Container>);
  });

  it('should render correctly', () => expect(wrapper).toMatchSnapshot());
});
