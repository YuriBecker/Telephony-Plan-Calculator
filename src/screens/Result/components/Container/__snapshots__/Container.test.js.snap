// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Container component should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#212121",
      "flex": 1,
      "justifyContent": "space-around",
      "padding": 30,
    }
  }
>
  Some data
</View>
`;
